### PR TITLE
APPIOS-2549: Use single endpoint for checking user's allowance

### PR DESCRIPTION
**Pull request by softwaresaiyajin**

### CODE REVIEW

File: Sources/PayseraAccountsSDK/Entities/PSAccountAuthorizationAllowance.swift [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import Foundation
+import ObjectMapper
+
+public class PSAccountAuthorizationAllowance: Mappable {
+    
+    public var accountNumber: String!
+    public var allowed: Bool
+    
+    required public init?(map: Map) {
```
Comment (Pattio) [inline Sources/PayseraAccountsSDK/Entities/PSAccountAuthorizationAllowance.swift:9]: Let's do mapping in `public func mapping(map: Map) {`, similarly to `PSBullion

File: Sources/PayseraAccountsSDK/Entities/PSClientAllowance.swift [hunk 2]
```diff
@@ -3,11 +3,19 @@ import ObjectMapper
...
 public class PSClientAllowance: Mappable {
     
-    public var allowed: Bool
+    public var userId: Int!
+    public var canOrderCard: Bool
+    public var canFillQuestionnaire: Bool
+    public var canCreateAccount: Bool
+    public var accountAuthorizationAllowances: [PSAccountAuthorizationAllowance]
     
     required public init?(map: Map) {
```
Comment (Pattio) [inline Sources/PayseraAccountsSDK/Entities/PSClientAllowance.swift:12]: Also `public func mapping(map: Map) {`

**Review by Pattio — CHANGES_REQUESTED** [review]

File: Sources/PayseraAccountsSDK/Entities/PSAccountAuthorizationAllowance.swift (hunk 1 above)
Comment (softwaresaiyajin) [reply Sources/PayseraAccountsSDK/Entities/PSAccountAuthorizationAllowance.swift]: Done

File: Sources/PayseraAccountsSDK/Entities/PSClientAllowance.swift (hunk 2 above)
Comment (softwaresaiyajin) [reply Sources/PayseraAccountsSDK/Entities/PSClientAllowance.swift]: Done

**Pull request merged**